config/locales/user_mailer/en.yml

```diff
@@ -44,6 +44,18 @@ en:
         %{help_link} or %{contact_link}.
       intro: The email address for your %{app} account has been changed.
     help_link_text: Help Center
+    letter_expired:
+      help: ''
+      info: The letter we sent to your address has a code that expired and can no
+        longer be used to complete the verification process.  Please sign into %{link}
+        and follow instructions.
+      subject: The letter we sent to your address has a code that has expired
+    letter_reminder:
+      help: ''
+      info: When you receive the letter it will contain a secret code.  Use this code
+        to complete the verification process by signing into %{link} and following
```

Comment:
Again, we could probably tell the user what they need to do here. "Verify your identity by signing into Login.gov and entering the code"